samples/bluetooth/hci_vs_scan_req/overlay-bt_ll_sw_split.conf

```diff
@@ -0,0 +1,3 @@
+CONFIG_BT_CTLR_VS_SCAN_REQ_RX=y
+CONFIG_BT_CTLR_ADVANCED_FEATURES=y
```

Comment:
Why do we need advanced features for this sample?

Reply:
It's a leftover from our application module, not needed here: removed.